File: app/OnyxPenDemo/src/main/java/com/onyx/android/eink/pen/demo/erase/data/EraseTypes.java
package com.onyx.android.eink.pen.demo.erase.data;

public final class EraseTypes {
    public static final int ERASER_STROKE = 0;
    public static final int ERASER_MOVE = 1;
    public static final int ERASER_AREA = 2;

    private EraseTypes() {}

    public static boolean isMoveOrStrokeErase(int eraseType) {
        return eraseType == ERASER_MOVE || eraseType == ERASER_STROKE;
    }
}

File: app/OnyxPenDemo/src/main/java/com/onyx/android/eink/pen/demo/erase/data/EraseTypes.kt
package com.onyx.android.eink.pen.demo.erase.data

object EraseTypes {
    const val ERASER_STROKE: Int = 0
    const val ERASER_MOVE: Int = 1
    const val ERASER_AREA: Int = 2

    fun isMoveOrStrokeErase(eraseType: Int): Boolean {
        return eraseType == ERASER_MOVE || eraseType == ERASER_STROKE
    }
}
